R-package/LICENSE.note

```diff
@@ -0,0 +1,84 @@
+
```

Comment:
Can we just do something like this:
```
#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 
#       Regularized Greedy Forest (RGF) 
#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 

See file COPYING: https://github.com/RGF-team/rgf/blob/master/RGF/COPYING

...
```
to avoid information duplication?

Reply:
fixed and pushed changes.

Reply:
Thank you for notifying, but you may not to waste your time typing this, I'll see pushed commits anyway. =)